### ISSUE TITLE
step parameter

### ISSUE THREAD

**Issue by sergiomena** [huochaitiantang/pytorch-deep-image-matting#37]
Hi @huochaitiantang , firstly thanks for the good job! 
I am trying the training of the model and everything works well, but I don't understand the meaning of the parameter "step". 
In the train.sh is set to -1 and in the train.py to 10. Which is the meaning of these numbers?
It seems is serve to influence the learning rate during training, but I don't understand how
Thanks in advance!
